lib/translator/translation.h

```diff
@@ -29,6 +29,11 @@ struct BeginResponse {
   uint32_t alloc_len;  // Defines size of buffer passed to Translation::Complete
 };
 
+struct CompleteResponse {
+  ApiStatus status;
+  scsi::Status scsi_status;  // Return value of library consumer functions
```

Comment:
Why does the caller need access to this?

Reply:
The QueueCommand SCSI function hook expects a status code as a return value. The consumer needs to have access to this value to return it from the calling function